Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/dialog/CwaDialogHelper.kt

```diff
@@ -0,0 +1,106 @@
+package de.rki.coronawarnapp.ui.dialog
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+
+/**
+ * Helper function that creates a CwaDialog and displays it.
+ * Usage:
+ *
+ * ```kotlin
+ * createDialog {
+ *     title("Title example")
+ *     message("Message example")
+ *     positiveButton("Positive button text") { foo1() }
+ *     negativeButton("Negative button text") { foo2() }
+ *     neutralButton("Neutral button text") { foo3() }
+ *     dismissAction { foo4() }
+ *     setCancelable(false)
+ *     setDeleteDialog(true)
+ *     setCustomView(R.layout.dialog_view)
+ * }
+ * ```
+ */
+fun Fragment.createDialog(
```

Comment:
should we keep the old name `display*` it is not just creating the dialog, it does show it too ?

Reply:
Yes, you're right. I had them in parallel and had to name this one something different.